flink/v1.16/flink/src/test/java/org/apache/iceberg/flink/SimpleDataUtil.java

```diff
@@ -134,6 +135,19 @@ public static DataFile writeFile(
       String filename,
       List<RowData> rows)
       throws IOException {
+    return writeFile(table, schema, spec, conf, location, filename, rows, null);
+  }
+
+  public static DataFile writeFile(
```

Comment:
nit: add doc?

Reply:
I could add it, but this is util for testing. Is it needed?

Reply:
Yes, it's not necessary. Just wondering if we could explain it briefly.